crypto/curve25519/curve25519.c

```diff
@@ -230,6 +230,9 @@ static void fe_1(fe h) {
   h[0] = 1;
 }
 
+/* Prototype to avoid -Wmissing-prototypes warnings. */
+void GFp_fe_add(fe h, const fe f, const fe g);
+
```

Comment:
Fixed (and now the prototype is for a _wrapper_ for the original `static` function).